binding/client.go

```diff
@@ -489,11 +498,19 @@ func (r *Client) FilePost(path, source string, object any) (err error) {
 // FilePut uploads a file.
 // Returns the created File resource.
 func (r *Client) FilePut(path, source string, object any) (err error) {
+	err = r.FilePutEncoded(path, source, object, "")
+	return
+}
+
+// FilePutEncoded uploads a file.
+// Returns the created File resource.
+func (r *Client) FilePutEncoded(path, source string, object any, encoding string) (err error) {
 	if source == "" {
 		fields := []Field{
 			{
-				Name:   api.FileField,
-				Reader: bytes.NewReader([]byte{}),
+				Name:     api.FileField,
+				Reader:   bytes.NewReader([]byte{}),
+				Encoding: encoding,
```

Comment:
Does the `Encoding` field need to get set on line 532 as well?

Reply:
Yes.  Good catch.  Fixed.